my-site.rb

```diff
@@ -2,7 +2,20 @@
 
 class MySite < Sinatra::Base
 
-  get "/" do
+  get "/:name" do
+    "Hello #{params[:name]}"
+  end
+
+  get "/hello.html" do
+    "Welcome to the INTERNET"
+#http://localhost:9292/hello.html
+ end
+
+  get "/candies" do
+    File.read('candies.html')
```

Comment:
You should be using the `erb :symbol` syntax rather than reading these files directly. See the example that you did together with charles